lib/src/command_runner.dart

```diff
@@ -73,19 +71,26 @@ class PubCommandRunner extends CommandRunner<int> implements PubTopLevel {
         return log.Verbosity.ALL;
       default:
         // No specific verbosity given, so check for the shortcut.
-        if (_argResults['verbose']) return log.Verbosity.ALL;
+        if (argResults['verbose']) return log.Verbosity.ALL;
         return log.Verbosity.NORMAL;
     }
   }
 
   @override
-  bool get trace => _argResults['trace'];
+  bool? get trace => argResults['trace'];
 
-  ArgResults _argResults;
+  ArgResults? _argResults;
```

Comment:
Perhaps make this a late final
```suggestion
  late final ArgResults argResults;
```